software/main.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/python3
+
+import time, sys
+
+import omni_motion
+from robot_utilities import *
+
+# Boot camp examples
+sys.path.append("./picr22-boot-camp-programming")
+import image_processor
+import camera
+
+# Open cv
+import cv2
+
+if __name__ == "__main__":
+    print("Starting...")
+    init_logging()
+    debug = False
+
+    # Setup from our code
+    max_speed = 10
+    max_speed_inner = 15
+    ball_good_range = 375
+    robot = omni_motion.Omni_motion_robot(max_speed_inner)
+    robot.start()
+
+    # Setup from example code
+    cam = camera.RealsenseCamera(exposure = 100)
+    processor = image_processor.ImageProcessor(cam, debug=debug)
+    processor.start()
+
+    # Constants etc.
+    # Housekeeping
+    start = time.time()
+    fps = 0
+    frame = 0
+    frame_cnt = 0
+    # Camera constants
+    middle_x = 424
```

Comment:
Would be more reasonable to calculate it based on current camera settings.

Reply:
Done.